nbclassic/static/style/style.less

```
@@ -27,9 +27,9 @@
 @import "../edit/less/style.less";
 
 // notebook
-@import "../notebook/less/style.less";
-@import "../notebook/less/commandpalette.less";
-@import "../notebook/less/searchandreplace.less";
+@import "../nbclassic/less/style.less";
+@import "../nbclassic/less/commandpalette.less";
+@import "../nbclassic/less/searchandreplace.less";
```

I also updated these lines to include the current path to these stylesheets like the previous change
```suggestion
@import "../notebook/less/style.less";
@import "../notebook/less/commandpalette.less";
@import "../notebook/less/searchandreplace.less";
```